services/hdbpp-cluster-reporting/server/app.py

```diff
@@ -220,6 +221,9 @@ def create_app(config_name):
     api.add_resource(attributes_endpoint.AttributesCurrentSize, '/database/tables/current_size/<string:att_format>/<string:att_type>')
     api.add_resource(attributes_endpoint.AttributesSizeUnit, '/database/tables/current_size/unit')
     
+    api.add_resource(aggregates_endpoint.AggregatesRowCount, '/database/aggregates/row_count/<string:agg_interval>/<string:att_type>')
```

Comment:
Just one more summary endpoint "/database/aggregates", that returns a dict of aggregates data tables mapped to a list of intervals available for that table. From this we can query the other endpoints with having to hard code anything,